docs/GettingStartedAriesDev/CredentialRevocation.md

```diff
@@ -64,9 +64,9 @@ These are the ACA-py steps and APIs involved to support credential revocation.
         "credential_exchange_id": credential_exchange_id
     }   
    ```
-0. Revoking credential
+0. Revoking credential (cred_rev_id is revocation_id from /issue-credential​/records)
     ```
-    POST /issue-credential/records/{cred_ex_id}/revoke
+    POST /issue-credential/revoke
```

Comment:
Could we add an example body for this POST request? I think this POST will need query params as
```
issue-credential/revoke?rev_reg_id=<revocation_registry_id>&cred_rev_id=<credential_definition_id>&publish=<true|false>
```

Reply:
![image](https://user-images.githubusercontent.com/21215409/130090146-e4f230e4-3b16-4106-8826-1e91458409f5.png)
 
 @swcurran 
 Please Check, the URL once, could not revoke credential in that URL. Image attaches along with it. Thanks!!